styles/globals.css

```diff
@@ -100,6 +100,10 @@ body,
   scrollbar-gutter: stable;
 }
 
+nav {
```

Comment:
instead of global class you can add `pl-[calc(100vw-100%)]` in the classname of nav element

Reply:
will do.

Reply:
@hemantwasthere Request you to please check now.